src/messages/MClientCaps.h

```diff
@@ -140,17 +135,7 @@ class MClientCaps : public Message {
 	      int dirty,
 	      int mseq,
               epoch_t oeb)
-    : Message(CEPH_MSG_CLIENT_CAPS, HEAD_VERSION, COMPAT_VERSION),
-      size(0),
-      max_size(0),
-      truncate_size(0),
-      change_attr(0),
-      truncate_seq(0),
-      time_warp_seq(0),
-      osd_epoch_barrier(oeb),
```

Comment:
This was accidentally removed. It's initializing `osd_epoch_barrier(0)` (the default). Causes: http://tracker.ceph.com/issues/23927